readme.md

```diff
@@ -106,8 +109,21 @@ Then, make each route respond with a one-word string containing the RESTful acti
 var express = require("express");
 var app = express();
 
-// Your code starts here...
+// app.post('/', function(req, res){
+// res.send('create')
+// });
+
+// app.get('/', function(req, res){
+// res.send('retrieve')
+// });
 
+// app.put('/', function(req, res){
+// res.send('update')
+// });
+
+// app.delete('/', function(req, res){
+// res.send('destroy')
+// });
```

Comment:
:+1: put and delete should have a `:id` in the path to know which one to update/delete